stronghold/middleware.py

```diff
@@ -1,8 +1,9 @@
 from django.contrib.auth.decorators import user_passes_test
+from django.utils.deprecation import MiddlewareMixin
 from stronghold import conf, utils
 
 
-class LoginRequiredMiddleware(object):
+class LoginRequiredMiddleware(MiddlewareMixin):
```

Comment:
This will only work for Django >= 1.10.  However, stronghold supports older versions of Django too. So, a solution could be subclass from object or MiddlewareMixin depending on Django version.